pilot/cmd/pilot-agent/options/options.go

```diff
@@ -75,6 +75,8 @@ var (
 
 	secretRotationGracePeriodRatioEnv = env.RegisterFloatVar("SECRET_GRACE_PERIOD_RATIO", 0.5,
 		"The grace period ratio for the cert rotation, by default 0.5.").Get()
+	workloadRSAKeySizeEnv = env.RegisterIntVar("WORKLOAD_RSA_KEY_SIZE", 2048,
```

Comment:
How is it going with an invalid value

Reply:
Container startup fails and therefore, the pod wouldn't be running.
But again, as mentioned earlier, this would be the case with most other envs right? It is the responsibility of the user to provide a valid value and not break the system.